internal/support/notifications/v2/controller/http/notification_test.go

```diff
@@ -39,7 +41,7 @@ var (
 	testNotificationSeverity    = models.Normal
 )
 
-var testAddNotificationRequest = dtos.Notification{
+var testAddNotification = dtos.Notification{
```

Comment:
Why does `Notification` not have a factory method to use here?

Reply:
Modified. Use the factory method to create a `Notification` DTO for testing.